app/models/compute_resource.rb

```diff
@@ -6,6 +6,12 @@ class ComputeResource < ActiveRecord::Base
   include Parameterizable::ByIdName
   encrypts :password
 
+  attr_accessible :name, :provider, :description,
+      :url, :display_type, :set_console_password, :zone, :project,
```

Comment:
* password should be an attr_protected
* uuid, ovirt quota etc.. should belong in their respective compute resources (you can see in https://github.com/dLobatog/foreman/commit/4d0dda5f662e265f73a2c81b90c5af34a1657435#diff-9bac1c9501e1b91fb9bfa5c8b9d251f3L11) to avoid figuring out where goes what
* There's attr_accessible in line 41, you might want to put that there